org.jellyfin.JellyfinServer.yml

```diff
@@ -214,29 +227,20 @@ modules:
       - -DBUILD_TESTS=OFF
       - -DFFT_LIB=fftw3f
     sources:
-      - type: archive
-        url: https://deb.debian.org/debian/pool/main/c/chromaprint/chromaprint_1.5.1.orig.tar.gz
-        sha256: a1aad8fa3b8b18b78d3755b3767faff9abb67242e01b478ec9a64e190f335e1c
-        x-checker-data:
-          type: html
-          url: https://packages.debian.org/source/bookworm/chromaprint
-          version-pattern: chromaprint \((\d+\.\d+\.\d+)
-          url-template: https://deb.debian.org/debian/pool/main/c/chromaprint/chromaprint_$version.orig.tar.gz
-      - type: archive
-        url: https://deb.debian.org/debian/pool/main/c/chromaprint/chromaprint_1.5.1-2.debian.tar.xz
-        sha256: e8fb8f5b309ce59a18b8d754f1ad8ada6a2246bbc8acbef47e5355c3b4fa3a1e
-        dest: debs
+      - type: git
+        url: https://github.com/acoustid/chromaprint.git
+        commit: 5c3be6803856f25b11d1d704bac56e733b4fc997
+        # NOTE: Pick the latest release, not the latest tag. The latter might not compile.
+        tag: v1.5.1
         x-checker-data:
-          type: html
-          url: https://packages.debian.org/source/bookworm/chromaprint
-          version-pattern: chromaprint \(([^\)]+)\)
-          url-template: https://deb.debian.org/debian/pool/main/c/chromaprint/chromaprint_$version.debian.tar.xz
-      - type: shell
-        commands:
-          - for p in $(cat debs/patches/series); do if [[ $p != "#"* ]]; then patch
-            -Np1 -i "debs/patches/$p" -d .; fi; done
+          type: git
+          tag-pattern: ^v([\d.]+)$
 
   - name: libopenmpt
+    buildsystem: simple
+    build-commands:
+      #- ./configure
+      - make -j$FLATPAK_BUILDER_N_JOBS PREFIX=$FLATPAK_DEST install
```

Comment:
drop buildsystem and the rest, add `no-autogen: true`

Reply:
I get this error message when I try your suggested changes:

```
install: cannot create directory ‘/usr/local’: Read-only file system
```

Reply:
Add PREFIX in `make-install-args`